evalytics/conftest.py

```diff
@@ -0,0 +1 @@
+# PATH issue with pytest: https://stackoverflow.com/a/50610630/1614677
```

Comment:
depending on the decision of the prev. ENV PYTHONPATH, this file needs to be located here *OR* at `evalytics/server` just choose one of them :)